lib/tpm2_eventlog_yaml.c

```diff
@@ -418,27 +418,49 @@ static bool yaml_uefi_var(UEFI_VARIABLE_DATA *data, size_t size, UINT32 type,
                 }
                 return true;
             }
-            /* Other variables will be printed as a hex string */
         } else if (type == EV_EFI_VARIABLE_AUTHORITY) {
-            free(ret);
-            tpm2_tool_output("    VariableData:\n");
-            
-            EFI_SIGNATURE_DATA *s= (EFI_SIGNATURE_DATA *)&data->UnicodeName[
-                data->UnicodeNameLength];
-            char *sdata = calloc (1,
-                BYTES_TO_HEX_STRING_SIZE(data->VariableDataLength - sizeof(EFI_GUID)));
-            if (sdata == NULL) {
-                LOG_ERR("Failled to allocate data: %s\n", strerror(errno));
-                return false;
+            /* The MokListTrusted is boolean option, not a EFI_SIGNATURE_DATA*/
+            if ((strlen(ret) == 14 && strncmp(ret, "MokListTrusted", 10) == 0)) {
```

Comment:
@THS-on can you please #define the magic numbers here

Reply:
@idesai I can do that but the checks in line 324f for `EV_EFI_VARIABLE_DRIVER_CONFIG` also don't do that.
Should I add defines then also for those in an extra commit?

Reply:
Yes, Thank you.

Reply:
Done